### PR TITLE
fix nullable fields in TripModel

### LINKED ISSUE #80: Fix the Trip model object on the server
At least ID and UserID MUST be nullable to allow the server to fill them in for the user.

### PR DESCRIPTION
**Pull request by cli1903**

### CODE REVIEW

**Review by anfee1 — DISMISSED** [review]

**Review by bsidhom — DISMISSED** [review]
LGTM

**Review by anfee1 — APPROVED** [review]

File: frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java [hunk 1]
```diff
@@ -9,9 +9,9 @@ public static Builder builder() {
     return new AutoValue_TripModel.Builder();
   }
 
-  public abstract String id();
-  @Nullable public abstract String name();
-  public abstract String userId();
+  @Nullable public abstract String id();
+  public abstract String name();
+  @Nullable public abstract String userId();
```
Comment (anfee1) [inline frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java:14]: We want the user id to be nullable?

**Review by anfee1 — APPROVED** [review]

File: frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java (hunk 1 above)
Comment (KoopaKing) [reply frontend/src/main/java/com/google/tripmeout/frontend/TripModel.java:14]: I don't think we expect the client to provide the userId as part of the request payload itself.

**Review by KoopaKing — APPROVED** [review]

**Review by bsidhom — APPROVED** [review]
LGTM

**Pull request merged**